Source/Extensions/Blazorise.DataGrid/DataGridSelectColumn.cs

```diff
@@ -8,27 +8,26 @@ public class DataGridSelectColumn<TItem> : DataGridColumn<TItem>
 {
     public override DataGridColumnType ColumnType => DataGridColumnType.Select;
 
-
-    //TODO : Introduce TSelectItem and TValue instead of boxing?
-
-
     /// <summary>
     /// Gets or sets the select data-source.
     /// </summary>
     [Parameter] public IEnumerable<object> Data { get; set; }
 
     /// <summary>
     /// Method used to get the display field from the supplied data source.
+    /// You can unbox the object by using the following example: TextField="(x) => ((Model)x).Text"
```

Comment:
A suggestion. When you have comments into new lines, maybe try to separate them with paragraphs. eg `<para></para>` .

Reply:
Usually do that, didn't remember in this PR. :)